Heap_Sort/Heap_Sort.php

```diff
@@ -0,0 +1,73 @@
+<?php
+  
+  /*
+    
+    Heap Sort is a comparison-based sorting technique performed on a Heap data structure
+        
+  */
+ 
+function MaxHeapify(&$data, $heapSize, $index) {
+    $left = ($index + 1) * 2 - 1; // left = 2*i + 1 
+    $right = ($index + 1) * 2; // right = 2*i + 2 
+    $largest = 0;
+
+    // To heapify a subtree rooted with node i which is 
+    // an index in arr[]. n is size of heap 
+     if ($left < $heapSize && $data[$left] > $data[$index])  // If left child is larger than root 
+         $largest = $left;
+     else
+         $largest = $index;
+ 
+     if ($right < $heapSize && $data[$right] > $data[$largest]) // If right child is larger than largest so far 
+         $largest = $right;
+  
+    if ($largest != $index)  // If largest is not root 
+   {
```

Comment:
The indentation of the if block is one less i.e 3 spaces. This bracket starts one space before if. Please add one more space before bracket also.